webrender/src/render_backend.rs

```diff
@@ -105,6 +105,59 @@ impl Document {
     }
 }
 
+struct WebGL {
+    last_id: WebGLContextId,
+    contexts: HashMap<WebGLContextId, GLContextWrapper, BuildHasherDefault<FnvHasher>>,
```

Comment:
Th WebGL code in WR will be deleted after https://github.com/servo/servo/pull/17891 lands, so it's not worth the effort to work on that.